rust/connlib/tunnel/src/client.rs

```diff
@@ -988,26 +988,34 @@ impl ClientState {
             return;
         }
 
-        match &new_resource {
+        let added = match &new_resource {
             ResourceDescription::Dns(dns) => {
-                self.stub_resolver.add_resource(dns.id, dns.address.clone());
+                self.stub_resolver.add_resource(dns.id, dns.address.clone())
             }
             ResourceDescription::Cidr(cidr) => {
                 let existing = self
                     .active_cidr_resources
                     .insert(cidr.address, cidr.clone());
 
                 match existing {
-                    Some(existing) if existing.id != cidr.id => {
-                        tracing::info!(address = %cidr.address, old = %existing.name, new = %cidr.name, "Replacing CIDR resource");
-                    }
-                    Some(_) => {}
-                    None => {
-                        tracing::info!(address = %cidr.address, name = %cidr.name, "Activating CIDR resource");
-                    }
+                    Some(existing) => existing.id != cidr.id,
+                    None => true,
                 }
             }
-            ResourceDescription::Internet(_) => {}
+            ResourceDescription::Internet(_) => false, // FIXME: Update with real check once Internet Resources get added.
+        };
+
+        if !added {
+            return;
+        }
+
+        let name = new_resource.name();
+        let address = new_resource.address_string();
+        let sites = new_resource.sites_string();
+
+        match address {
+            Some(address) => tracing::info!(%name, %address, %sites, "Activating resource"),
+            None => tracing::info!(%name, %sites, "Activating resource"),
```

Comment:
Alternative to the above comment
```suggestion
            None => tracing::info!(%name, type = "Internet", %sites, "Activating resource"),
```

But something more elegant than that

Reply:
`name` will already say "Internet" :)